app/css/custom.scss

```diff
@@ -871,4 +947,4 @@ blockquote {
       }
     }
   }
-}
+}
```

Comment:
The reason why this was done like this was because there is no particular structure for the sass files. The structured sass files is the main.scss which was not included in  the import statement on the main Zhishi component. But am updating asap to something better